src/ucp/wireup/wireup.c

```diff
@@ -688,20 +692,15 @@ static ucs_status_t ucp_wireup_msg_handler(void *arg, void *data,
     ucp_worker_h worker   = arg;
     ucp_wireup_msg_t *msg = data;
     ucp_unpacked_address_t remote_address;
-    ucp_ep_h ep UCS_V_UNUSED;
     ucs_status_t status;
 
     UCS_ASYNC_BLOCK(&worker->async);
 
     if (msg->dst_ep_id != UCP_EP_ID_INVALID) {
-        ep = ucp_worker_get_ep_by_id(worker, msg->dst_ep_id);
-        if (ep == NULL) {
-            ucs_diag("got wireup msg %d src_ep_id 0x%"PRIx64" for"
-                     " non-existing dst_ep_id 0x%"PRIx64" sn %d,"
-                     " ignoring it",
-                     msg->type, msg->src_ep_id, msg->dst_ep_id, msg->conn_sn);
-            goto out;
-        }
+        UCP_WORKER_GET_EP_BY_ID(worker, msg->dst_ep_id, 0, goto out,
```

Comment:
can ep bein failed state here?

Reply:
yes, but it also can be in closed state and we have to wait for WIREUP procedure completion, otherwise - flush won't be completed, since UCT lanes are WIREUP EPs